src/annotator/plugin/test/pdf-test.js

```diff
@@ -45,7 +45,7 @@ describe('annotator/plugin/pdf', () => {
     };
 
     $imports.$mock({
-      './pdf-metadata': () => fakePDFMetadata,
+      './pdf-metadata': sinon.stub().returns(fakePDFMetadata),
```

Comment:
Could you explain this one to me like I'm 5? I want to understand how the arrow-bound variant fails. It would probably make sense to me if I saw it break the way you did initially.

Reply:
You can see the problem in a browser console:

<img width="505" alt="arrow-function" src="https://user-images.githubusercontent.com/2458/96586929-9318a080-12d9-11eb-958d-08d7b3b4fb9b.png">

The latter syntax is what arrow function used to get turned into with the old Babel settings. Now that no longer happens.